packages/trpc/server/routers/viewer/workflows/util.ts

```diff
@@ -658,8 +659,33 @@ export async function scheduleBookingReminders(
             sendTo = [step.sendTo || ""];
             break;
         }
+
+        const organizerT = await getTranslation(bookingInfo.organizer.language.locale, "common");
+
+        const attendeePromises = [];
+        for (const attendee of bookingInfo.attendees) {
+          attendeePromises.push(
+            getTranslation(attendee.language.locale ?? "en", "common").then((tAttendee) => ({
+              ...attendee,
+              language: { translate: tAttendee, locale: attendee.language.locale ?? "en" },
+            }))
+          );
+        }
+
+        const attendees = await Promise.all(attendeePromises);
+
+        const emailEvent = {
+          ...bookingInfo,
+          type: bookingInfo.eventType.slug!,
+          organizer: {
+            ...bookingInfo.organizer,
+            language: { ...bookingInfo.organizer.language, translate: organizerT },
+          },
+          attendees,
+        };
```

Comment:
do we need to do that in other places too where we call `scheduleEmailReminder`?